### DISCUSSION TITLE
h with two ascenders???

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#134] [General]
The question is slightly related to #133. "Polonia Typographica" says the character was used by several printers, but I found only the enclosed occurences in a text:
![Rubricella_ephia_context](https://user-images.githubusercontent.com/8024702/165234538-744326e1-0cc7-4c9c-9e93-c42771113d03.png)
The character is not included in MUFI (unless I missed something) and I found no information about it on the Internet (although a similar character was used in Gutenberg's Bible but, if I understand correctly, for purely decorative purposes). So I asked a question on the Facebook The Paleography Society group and Lisa Howarth just wrote 

> When attached to an 'h', it usually means 'er' or 'ab' depending on the word.

So for her this is an h with a diacritics.

**Comment by psb1558** [comment]
For Epiphania, I guess?

You can get very close to what you see here with Junicode: h + U+0335 with cv16[1]:
<img width="223" alt="image" src="https://user-images.githubusercontent.com/16215767/165280017-915f80e7-0982-44d7-9409-ad296ed5ab31.png">

The flourish on the right end of the horizontal stroke can be added to any horizontal stroke (in MSS and early prints). If it's needed I could make it a variant of U+0335.

**Reply by jsbien** [reply]
Yes, I was told it is "Epiphania".
I don't need the character now, I was just curious. Hence your solution is OK for me.
